services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
+
+    return [{"row": ticket.row, "seat": ticket.seat} for ticket in tickets]
```

Comment:
you can get result in one line and use `.values()`

Reply:
still not fixed